If you use -fpic instead of -fPIC, linker fall down, go boom.
--- apache_1.3.14/src/Configure.fpic	Fri Dec  1 15:25:17 2000
+++ apache_1.3.14/src/Configure	Fri Dec  1 15:25:42 2000
@@ -1034,14 +1034,14 @@
 	    LDFLAGS_MOD_SHLIB=$LDFLAGS_SHLIB
 	    ;;
 	*-linux1)
-	    CFLAGS_SHLIB="-fpic"
+	    CFLAGS_SHLIB="-fPIC"
 	    LDFLAGS_SHLIB="-Bshareable"
 	    LDFLAGS_MOD_SHLIB=$LDFLAGS_SHLIB
 	    LDFLAGS_SHLIB_EXPORT="-rdynamic"
 	    ;;
 	*-linux2)
 		LD_SHLIB="gcc"
-	    CFLAGS_SHLIB="-fpic"
+	    CFLAGS_SHLIB="-fPIC"
 	    LDFLAGS_SHLIB="-shared"
 	    LDFLAGS_MOD_SHLIB=$LDFLAGS_SHLIB
 	    LDFLAGS_SHLIB_EXPORT="-rdynamic"
